### DISCUSSION TITLE
Node.js environment not being set in Dockerfile

### DISCUSSION THREAD

**Discussion by jorisvervuurt** [Koenkk/zigbee2mqtt#14625] [General]
Maybe I’m missing something, but it seems that the Node.js environment is not being set.

Perhaps the following needs to be added to the Dockerfile, just above the `CMD` command?

```
ARG NODE_ENV=production
ENV NODE_ENV $NODE_ENV
```

This might improve general performance:<br>https://nodejs.dev/en/learn/nodejs-the-difference-between-development-and-production/

**Comment by Koenkk** [comment]
It was not set, added it for both the z2m and z2m addon container!

Bedankt 😄
